ingestion/assign_cadet_databases_source/source.py

```diff
@@ -0,0 +1,76 @@
+import json
+from typing import Dict, Iterable
+import logging
+
+import datahub.emitter.mce_builder as mce_builder
+import datahub.emitter.mcp_builder as mcp_builder
+from datahub.ingestion.api.common import PipelineContext
+from datahub.ingestion.api.decorators import config_class
+from datahub.ingestion.api.source import Source, SourceReport
+from datahub.ingestion.api.workunit import MetadataWorkUnit
+
+from ingestion.config import ENV, INSTANCE, PLATFORM
+from ingestion.assign_cadet_databases_source.config import (
+    AssignCadetDatabasesConfig,
+)
+from ingestion.create_derived_table_databases_source.source import \
+    get_cadet_manifest
+
+
+@config_class(AssignCadetDatabasesConfig)
+class AssignCadetDatabases(Source):
```

Comment:
I think at the point where we assign the model to the container, we should also propagate any `dc_display_in_catalogue` tags from the model to the container. Otherwise they won't show up in the frontend.

Alternatively we could set `dc_display_in_catalogue` on all the containers when we create them, but I think that would expose some databases we are not intending to catalogue.

Reply:
I've made a ticket for this. https://github.com/ministryofjustice/data-catalogue/issues/152